src/components/TableLine/TableLine.jsx

```diff
@@ -1,15 +1,15 @@
 import PropTypes from 'prop-types';
 
 export const TableLine = ({ number, name, children, ...restProps }) => {
-  const propsOtherFiels = Object.entries(restProps);
+  const propsOtherFiedls = Object.entries(restProps);
 
   return (
     <div className='tableLine'>
       <div className='tableLine__number tableLine-elem'>{number}</div>
       <a href='/' className='tableLine__name tableLine-elem'>
         {name}
       </a>
-      {propsOtherFiels.map(([key, value]) => (
+      {propsOtherFiedls.map(([key, value]) => (
```

Comment:
...Fields

Reply:
thx for attentiveness, it's so ashamed for me
```suggestion
      {propsOtherFields.map(([key, value]) => (
```